src/plugins/console/index.js

```diff
@@ -37,7 +54,8 @@ module.exports = function (kibana) {
     config: function (Joi) {
       return Joi.object({
         enabled: Joi.boolean().default(true),
-        defaultServerUrl: Joi.string().default('http://localhost:9200'),
+        defaultServerUrl: Joi.string().optional(),
+        proxyTargets: Joi.array().items(Joi.string()).optional(),
```

Comment:
@spalger Should we allow users to specify a single `proxyTarget` in the config like so: `console.proxyTarget: http://foobar` (as opposed to requiring them to wrap that single value in an array like so: `console.proxyTarget: [ http://foobar ]`)?